dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/StaticLinkageCheckOption.java

```diff
@@ -38,7 +38,7 @@
  *
  * <ul>
  *   <li>{@code bomCoordinates}: Maven coordinates for a BOM
- *   <li>{@code mavenCoordinates}: list of the coordinates of Maven artifacts to check
+ *   <li>{@code mavenCoordinatesList}: list of the coordinates of Maven artifacts to check
```

Comment:
This is a small thing, but in general you don't need to name methods or parameters with "list" if they're of a List type. So-called ["Hungarian notation"](https://en.wikipedia.org/wiki/Hungarian_notation) isn't useful in a statically typed language.

This is why I often don't bother with "list of" phrases in documentation either.

It also makes it easier to switch between lists and sets later on, since that doesn't change the meaningful semantics at the documentation level.